code-quality/checkstyle/checkstyle-suppressions.xml

```diff
@@ -8,6 +8,8 @@
     <suppress files=".*Application.java" checks="HideUtilityClassConstructor"/>
 
     <suppress files="[\\/]model[\\/].*.java" checks="JavadocMethod"/>
+    <suppress files="[\\/]config[\\/].*.java" checks="JavadocMethod"/>
```

Comment:
Yes, it is an example how it works. I asked @andrei-astaschenko earlier to add such documentation, I think he does it better as he implemented this functionality. TODO will be added